docs/source/supercheq/Supercheq_qss.ipynb

```diff
@@ -5,73 +5,113 @@
    "id": "7769945b",
```

Comment:
<p>Update to <code>##</code> instead of <code>#</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>